style.css

```diff
@@ -1,4 +1,312 @@
+  font-weight: 500;
+  display: inline-block;
+  height: 44px;
+  margin-right: 53px;
+  position: relative;
+  text-transform: uppercase;
+  transform: translateX(-1px);
+  width: 203px;
+}
+
+.range {
+  display: inline-block;
+  position: relative;
+}
+
+.range__link {
+  color: #000;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 500;
+  display: inline-block;
+  margin-right: 17px;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 172px;
+}
+
+.rangle__link-arrow {
+  display: inline-block;
+}
+
+.socials-list {
+  display: flex;
+  justify-content: space-between;
+  height: 43px;
+  margin-left: 52px;
+  margin-top: 155px;
+  padding: 0;
+  width: 160px;
+ }
+
+.socials__item-pic {
+  height: 43px;
+  margin-right: 8px;
+  width: 43px;
+}
+
+.scroll {
+  display: flex;
+  height: 46px;
+  margin-bottom: 18px;
+  margin-left: 707px;
+  transform: translateY(-42px);
+  width: 27px;
+}
+
+.scroll__btn {
+  background-color: white;
+  box-sizing: border-box;
+  border-radius: 15px;
+  cursor: pointer;
+  position: relative;
+  z-index: 1;
+}
+
+.scroll__wheel { 
+  background: #000;
+  border-radius: 19px;
+  height: 8px;
+  display: inline-block;
+  margin-top: 10px;
+  width: 3px;
+  transform: translateX(-15px);
+  z-index: 2;
+}
+
+@media (min-width: 375px) and (max-width: 480px) {
+
+  * {
+  	box-sizing: border-box;
+  }
+
+  html,
+  body {
+  	margin: 0 auto;
+  	max-width: 480px;
+  	min-width: 375px;
+  	overflow-x: hidden;
+  	width: 100%;
+  }
+
+  .header,
+  .collection {
+    background: none;
+    font-size: 0;  
+    margin: 0;
+    width: 100%;
+  }
+
+  .header {
+    background-color: #fff;
+    display: flex;
+    font-size: 0;
+    height: 69px;
+    justify-content: flex-start;  
+ }
+
+  .header__logo {
+    background-image: url('./images/LOCO.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: fixed;
+  	display: flex;
+    height: 37px;
+    margin-left: 23px;
+    transform: translateY(-2px);
+    width: 98px;
+  }
+
+  .header-menu {
+  	background-color: #ededed;
+  	height: 69px;
+  	margin: 0;
+  	right: 0;
+  	width: 70px;
+  }
+
+  .header-menu__line::before {
+    content: '';
+    transform: translateY(-9px);
+  }
+
+  .header-menu__line,
+  .header-menu__line::before {
+  	display: block;
+  	height: 3px;
+  	left: 50%;
+    margin-left: -14px;
+    top: 52%;
+  	width: 27px;
+  }
+
+  .language,
+  .collection__img,
+  .collection__description,
+  .collection__btn,
+  .range,
+  .socials-list,
+  .scroll {
+  	display: none;
+  }
+
+  .collection {
+    background-image: url('./images/pete-bellis-432867-unsplash-mob.png');
+    background-position: center bottom;
+    background-size: cover;
+    min-height: 529px;
+    object-fit: cover;
+    width: 100%;
+  }
+
+  .collection__title {
+  	display: inline-block;
+  	font-size: 50px;
+    font-style: normal;
+    font-weight: bold;
+  	height: 103px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)